src/Uno.UI.FluentTheme/Uno.UI.FluentTheme.csproj

```diff
@@ -38,6 +38,13 @@
 		<ProjectReference Include="..\Uno.UI\Uno.UI.csproj" />
 	</ItemGroup>
 	
+	<ItemGroup Condition="$(IsMonoAndroid) or $(IsXamarinIOS) or $(IsXamarinMac)">
+		<Reference Include="System.Numerics" />
+		<Reference Include="System.Numerics.Vectors" />
+		<Reference Include="System.json" />
```

Comment:
capitalization looks off?

Reply:
Actually copied the snippet from Uno.UI.csproj 😀 . Will fix in both then